config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -0,0 +1,67 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
The entire file is commented, do we need to commit it?

Reply:
https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3753#discussion_r655476724

Reply:
I'd be inclined to keep  - to [quote this blog](http://joshfrankel.me/blog/a-guide-for-upgrading-to-rails-6/)
>This is a new convention from Rails 5 that lists out specific feature flags that can be opted into at a later date allowing for an easier time migrating to new Rails’ versions. Keep these as either commented out or false. Enabling these should be places as individual stories within the post-upgrade category.

Reply:
This [SO thread](https://stackoverflow.com/questions/50102584/upgrading-rails-what-am-i-to-do-with-new-framework-defaults-file/51275459) has a good description. I think we need to have a post upgrade ticket or tickets to flip these options 

The file itself contains comments for some like
>This change is not backwards compatible with earlier Rails versions.
It's best enabled when your entire app is migrated and stable on 6.1.

Reply:
Got it, I will mark these comments as resolved.